Release 2.1

rartych opened the pull request:
#### What type of PR is this?

* subproject management

#### What this PR does / why we need it:

This is Release PR for release 2.1 containing version v0.5.0-alpha.1 of Commonalities


#### Which issue(s) this PR fixes:

Fixes #340 


#### Does this PR introduce a breaking change?

- [x] Yes, some of the included PR in the release are having breaking changes
- [ ] No

<!-- If indicated yes above, please describe the breaking change(s). -->

#### Special notes for reviewers:



#### Changelog input

```
r2.1 containing version v0.5.0-alpha.1 of Commonalities

```

#### Additional documentation

rartych committed:
Update CHANGELOG.md

rartych committed:
Update README.md

rartych committed:
Merge branch 'camaraproject:main' into release-2.1

rartych committed:
Merge branch 'camaraproject:main' into release-2.1

rartych committed:
Update VERSION.yaml

rartych committed:
Update CHANGELOG.md

rartych committed:
Update CAMARA_common.yaml

rartych committed:
Update event-subscription-template.yaml

rartych committed:
Update notification-as-cloud-event.yaml

rartych committed:
Update CHANGELOG.md

rartych committed:
Update CHANGELOG.md

rartych committed:
Update CHANGELOG.md

rartych commented:
The description of PR# https://github.com/camaraproject/Commonalities/pull/329 will be added before final review

rartych committed:
Update CHANGELOG.md

rartych commented:
Ready for final review

PedroDiez commented on CHANGELOG.md:
```diff
@@ -1,14 +1,52 @@
+  - added rules for subscriptions with device identifier attribute
+  - added section 11.7 Resource access restriction relevant to subscriptions
+  - added clarification on `expiresAt` attribute for subscription
+* Updated error codes and changed `info.description` template for device / phone number identifiers in Appendix A in API Design Guideliness by @eric-murray in https://github.com/camaraproject/Commonalities/pull/324
```
Suggestion:

* Updated error codes and changed `info.description` template for device / phone number identifiers in Appendix A in API Design Guideliness by @eric-murray in https://github.com/camaraproject/Commonalities/pull/324 and https://github.com/camaraproject/Commonalities/pull/346

rartych replied:
resolved in https://github.com/camaraproject/Commonalities/pull/342/commits/acfbc56e8025ea10b4294f310a79637c2ecd0503

rartych committed:
Update CHANGELOG.md

Fixed typos and suggestions
Added https://github.com/camaraproject/Commonalities/pull/351

PedroDiez commented on CHANGELOG.md:
```diff
@@ -24,20 +24,20 @@ This release contains documents and artifacts of Commonalities version 0.5.0-alp
 ### Changed
 * Normalization of error status and code allowed values using `enum` by @PedroDiez in https://github.com/camaraproject/Commonalities/pull/316
-* Guidelines for subscription and event notification in API Design Guideliness 0.5.0 by @bigludo7 in https://github.com/camaraproject/Commonalities/pull/313 main changes:
+* Guidelines for subscription and event notification in API Design Guideliness by @bigludo7 in https://github.com/camaraproject/Commonalities/pull/313 main changes:
```
```suggestion
* Guidelines for subscription and event notification in API Design Guidelines by @bigludo7 in https://github.com/camaraproject/Commonalities/pull/313 main changes:
```

PedroDiez commented on CHANGELOG.md:
```diff
@@ -1,14 +1,52 @@
+  - added section 11.7 Resource access restriction relevant to subscriptions
+  - added clarification on `expiresAt` attribute for subscription
+* Updated error codes and changed `info.description` template for device / phone number identifiers in Appendix A in API Design Guideliness by @eric-murray in https://github.com/camaraproject/Commonalities/pull/324 and https://github.com/camaraproject/Commonalities/pull/346
+* Guidelines regarding mandatory error `status` and alignment of error codes related to identifiers in API Design Guideliness by @PedroDiez in https://github.com/camaraproject/Commonalities/pull/329 and https://github.com/camaraproject/Commonalities/pull/351
```
```suggestion
* Guidelines regarding mandatory error `status` and alignment of error codes related to identifiers in API Design Guidelines by @PedroDiez in https://github.com/camaraproject/Commonalities/pull/329 and https://github.com/camaraproject/Commonalities/pull/351
```

PedroDiez reviewed (commented):
Minor points

rartych committed:
Apply suggestions from code review

Co-authored-by: Pedro Díez García <pedro.diezgarcia@telefonica.com>

PedroDiez reviewed (approved):
LGTM

hdamker reviewed (approved):
LGTM - thanks for all the work.

For some of the listed PRs I'm not sure if the statement "no breaking change" is correct (from perspective of impacted API specifications). Have added comments there.

patrice-conil reviewed (approved):
LGTM
